app/controllers/catalog_controller.rb

```diff
@@ -161,9 +160,12 @@ class CatalogController < ApplicationController
       }
     }
 
-    config.add_results_collection_tool(:report_view_toggle)
     config.add_results_collection_tool(:bulk_update_view_button)
     config.add_results_collection_tool(:bulk_action_button)
+    config.add_results_collection_tool(:sort_widget)
+    config.add_results_collection_tool(:per_page_widget)
+    # config.add_results_collection_tool(:view_type_group)
```

Comment:
Cruft?

Reply:
This is a default that blacklight 7 generates into the config, so I thought we should leave it in case we want to add that tool back.